<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADRES_EMAIL_FISI</class>
 <widget class="QDialog" name="ADRES_EMAIL_FISI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>E-MAİL ADRESLERİ</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,60" columnstretch="20,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="kernel_buttons_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_adres_kaydi">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget_emailler"/>
   </item>
  </layout>
  <action name="action_satir_ekle">
   <property name="text">
    <string>Satır Ekle</string>
   </property>
  </action>
  <action name="action_satir_sil">
   <property name="text">
    <string>Satır Sil</string>
   </property>
  </action>
  <action name="action_satiri_tamamla">
   <property name="text">
    <string>Satiri Tamamla</string>
   </property>
   <property name="toolTip">
    <string>Satırdaki eksik bilgileri tamamlar.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableWidget_emailler</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
